getting-started/run-queries-from-console.md

```diff
@@ -61,11 +61,13 @@ To connect to a service:
 
 1. **Connect to your service**
 
-   1. In the [data mode][portal-data-mode] in Timescale Console, select a service and enter your password, then click **Connect**.
+   1. In the [data mode][portal-data-mode] in $CONSOLE, select a service in the connection drop-down. 
+   1. Click the pencil icon next to the service name. 
+   1. Click `Edit` next to `Username/Password` and enter the credentials for your service, then click `Connect`.
 
       ![Select a connection](https://assets.timescale.com/docs/images/data-mode-connections.png)
```

Comment:
Can you move the screenshot up one step, as it shows the pencil, not the Edit window.